src/header.jsx

```diff
@@ -204,15 +191,21 @@ const UploadButton = ({ files, setChunksProgress, isUploading, setIsUploading })
 
     return (
         <>
-            <Button
-              variant="secondary" onClick={handleClick}
-              isDisabled={isUploading}
-            >Upload
-            </Button>
+            <Button variant="secondary" onClick={handleClick}>Upload</Button>
             <input
               ref={ref} type="file"
               hidden onChange={onUpload}
             />
         </>
     );
 };
+
+const UploadProgress = ({ chunksProgress }) => {
+    const progress = Math.round(100 * chunksProgress.completed / chunksProgress.number);
+    return (
+        <div
+          id="progress" className="progress-pie"
+          title="Upload 22% completed" style={{ "--progress": `${progress}%` }}
```

Comment:
Don't forget to update the title to the correct percentage. Also, it should be "Upload: 22% complete" (with 22% the current progress percentage).